File: Python/BinaryTreePostOrderTraversal.py
class TreeNode:
    def __init__(self,val=0,left=None,right=None) -> None:
        self.val = val
        self.left = left
        self.right = right


class Solution:
    def postOrderTraversal(root):
        outputList = []


        def helper(node, outputList):
            if not node:
                return None
            
            helper(node.left,outputList)
            helper(node.right,outputList)
            outputList.append(node.val)


        helper(root,outputList)

        return outputList

File: Javascript/BinaryTreePostOrderTraversal.js
function TreeNode(val,left,right){
    this.val = (val === undefined) ? 0:val
    this.left = (left === undefined) ? null:left
    this.right = (right === undefined) ? null:right
}


var postOrderTraversal = function(root){
    const outputArray = []


    function helper(node){
        if(node === null){
            return null
        }

        helper(node.left)
        helper(node.right)
        outputArray.push(node.val)
    }

    helper(root)

    return outputArray
}